docs/installation.rst

```diff
@@ -20,18 +14,11 @@ Xonsh can be installed via package manager, via appimage or just run from a pre-
     packages
     appimage
     containers
-    editors
     jupyter
 
-Configuration and Customization
-================================
```

Comment:
This heading separation is useful

Reply:
Here I'm not removing only header. I remove this section and move the 3 links from it to the "Config Files and Settings" sections. It's consistent.

Reply:
Oh I see, thanks!